public/firmwares/manifest.json

```diff
@@ -15,19 +15,19 @@
   "firmwares": [
     {
       "name": "Zigbee (EZSP)",
-      "url": "/firmwares/skyconnect_ncp-uart-hw_7.4.3.0.gbl",
+      "url": "/firmwares/skyconnect_ncp-uart-hw_7.4.4.0.gbl",
       "type": "ncp-uart-hw",
-      "version": "7.4.3.0"
+      "version": "7.4.4.0"
     },
     {
       "name": "Multi-PAN (RCP)",
-      "url": "https://raw.githubusercontent.com/NabuCasa/silabs-firmware/main/RCPMultiPAN/beta/NabuCasa_SkyConnect_RCP_v4.3.2_rcp-uart-hw-802154_460800.gbl",
+      "url": "https://raw.githubusercontent.com/NabuCasa/silabs-firmware/main/RCPMultiPAN/beta/NabuCasa_SkyConnect_RCP_v4.4.0_rcp-uart-hw-802154_460800.gbl",
```

Comment:
Multi-PAN is locked at 4.3.2 and we aren't going to be updating it for the foreseeable future, despite the firmware builder releases containing newer firmwares.

Reply:
Thanks for letting me know. I did not know this. Is this fact documented somewhere (would be also interested in the reason)? If yes, can you point me to that documentation?

(If we would use JSON5 we could add a comment to the `manifest.json` so future version bumpers don't make the same mistake. Alternatively we could add a special key to that object like `"_doc": "This version is pinned and should never be changed. See <doc link>"`. Not sure if this would break the frontend though (haven't checked).

I amended my commit to exclude the Multi-PAN changes. :)

Thanks for your help on my PR!